app/js/redux-store.jsx

```diff
@@ -7,6 +7,7 @@
  * graphic logo is a trademark of OpenMRS Inc.
  */
 
+/* eslint import/no-extraneous-dependencies: ["error", {"peerDependencies": true}] */
```

Comment:
Do you still need the above line?

Reply:
yes, it sets this rule for only this file instead of the whole project.